api/src/main/java/org/openmrs/module/fhir/api/ConditionService.java

```diff
@@ -16,9 +16,11 @@
 import ca.uhn.fhir.rest.server.exceptions.NotModifiedException;
 import ca.uhn.fhir.rest.server.exceptions.ResourceNotFoundException;
 import org.hl7.fhir.dstu3.model.Condition;
+import org.springframework.transaction.annotation.Transactional;
 
 import java.util.List;
 
+@Transactional
```

Comment:
Our convention is to put the transactional annotation on the implementation instead of interface. https://tickets.openmrs.org/browse/TRUNK-208